config/overlays/acceptance/vault.yaml

```diff
@@ -24,9 +24,11 @@ metadata:
   name: vault
   namespace: vault
 spec:
+  type: NodePort
```

Comment:
Expose the vault service onto your local machine. You can now go to localhost:8200 when kind is running to access vault.